exporters/etw/include/opentelemetry/exporters/etw/etw_data.h

```diff
@@ -0,0 +1,536 @@
+#pragma once
+
+#include <cstdint>
+#include <cstdio>
+#include <cstdlib>
+
+#include <mutex>
+
+#include "opentelemetry/nostd/shared_ptr.h"
+#include "opentelemetry/nostd/string_view.h"
+#include "opentelemetry/nostd/unique_ptr.h"
+
+#include "opentelemetry/common/key_value_iterable_view.h"
+#include "opentelemetry/trace/span.h"
+#include "opentelemetry/trace/span_context_kv_iterable_view.h"
+#include "opentelemetry/trace/span_id.h"
+#include "opentelemetry/trace/trace_id.h"
+#include "opentelemetry/trace/tracer_provider.h"
+
+#include "opentelemetry/sdk/trace/exporter.h"
+#include "opentelemetry/sdk/trace/recordable.h"
+#include "opentelemetry/sdk/trace/span_data.h"
+
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <memory>
+#include <sstream>
+#include <unordered_map>
+
+#include "opentelemetry/exporters/etw/etw_provider_exporter.h"
+
+#include "opentelemetry/exporters/etw/utils.h"
+
+namespace core  = opentelemetry::core;
+namespace trace = opentelemetry::trace;
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+
+namespace ETW
```

Comment:
Usually we have exporters under `opentelemetry::exporter` namespaces.

Reply:
Yes.